psignifit/_result.py

```diff
@@ -18,7 +18,7 @@ def default(self, obj):
 
 @dataclasses.dataclass
 class Result:
-    parameter_estimate: Dict[str, float]
+    parameter_fit: Dict[str, float]
```

Comment:
what's the idea behind this name change?

Reply:
Easier to recognize the content for switcher